frontend/src/component/common/BreadcrumbNav/BreadcrumbNav.tsx

```diff
@@ -55,9 +55,15 @@ const BreadcrumbNav = () => {
         .map(decodeURI);
 
     if (location.pathname === '/insights') {
+        // Because of sticky header in Insights
```

Comment:
how is it related to archive?

Reply:
Comment was missing. I'll remember to add this kind of fixes in separate PRs in the future